README.md

```diff
@@ -141,6 +141,33 @@ Digest: <encryption verification>
 Status: Downloaded pega-docker.downloads.pega.com/platform/pega:<version>
 ```
 
+For details, see the examples listed in the preparation runbook:
+
+* [Preparing your local Linux system](docs/prepping-local-system-runbook-linux.md)
+* [Preparing your local Windows 10 system](docs/prepping-local-system-runbook-windows.md)
+
+Pegasystems uses a standard naming practice of hostname/product/image:tag. Pega images are available from the host site, pega-docker.downloads.pega.com. Pega maintains four types of required Docker images for Client-managed Cloud deployments of Pega Platform:
+
+ Name                                           | Description                                           | Tags     |
+------------------------------------------------|-------------------------------------------------------|----------|
+`platform/installer`                            | A utility image with which you install all of the Pega-specific rules and database tables in the “Pega” database that you have configured for your deployment. This installation is required before a deployment can take place.| `<version>` |
```

Comment:
this list of images is missing the Hazelcast image>>platform/clustering-service

This was the detail added by Varun's team a while ago...

`platform/clustering-service`  | (Download required) Deploys the required clustering service(Hazelcast) for the caching capabilities in Pega platform as a backing service. This Docker image contains Hazelcast clustering Service. <br/><br/>*Note: This feature is supported only from platform version "8.6" to deploy the platform in client-server Hazelcast model. Embedded deployment would not be supported in future platform releases.*| `<version>` |